GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/Utils/ServerURLs.cs

```diff
@@ -0,0 +1,7 @@
+public static class ServerURLs{
```

Comment:
if it is only used in one class, you can consider to move this into that class

Reply:
The class is use by 2 controllers.

Reply:
Only "NetworkRequestController". Why would another controller use this directly? All network requests shall be delegated to NetworkRequestController, right?